test/schema/avert.test.ts

```diff
@@ -0,0 +1,99 @@
+import { AvertRecord } from "@/schema/avert";
+import { PowerPlantClass, Location } from "@/schema/egrid";
```

Comment:
Generally, we want a unit test file to only test the source file that it's named after. We don't need to test `PowerPlantClass` and `Location` since it's already being tested in the eGRID schema tests